Dockerfile

```diff
@@ -1,3 +1,7 @@
 FROM nginx:1.15-alpine
 COPY dist/hypertrace-ui /usr/share/nginx/html
-COPY conf/default.conf /etc/nginx/conf.d/default.conf
\ No newline at end of file
+COPY conf/default.conf /etc/nginx/conf.d/default.conf
+# expose port for UI
+EXPOSE 2020
+# Healthcheck to check if client can connect to UI port
```

Comment:
nit: this comment and the one above it add no value if you look at them carefully!

Reply:
makes sense. removing it!